browser/ntp_background/ntp_p3a_helper_impl_unittest.cc

```diff
@@ -71,6 +71,8 @@ class NTPP3AHelperImplTest : public testing::Test {
         &local_state_, p3a_service_.get(), &prefs_, true);
   }
 
+  void TearDown() override { p3a_service_ = nullptr; }
```

Comment:
why p3a service is not recreated in SetUp() ?

Reply:
it is